.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - jrb/bootstrap
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        terraform_version: ["1.1.6"]
```

Comment:
Adding a version of Terraform to this matrix won't publish a new variant of this image by itself. We need to maintain a Nix flake for each version of Terraform we want to publish.